template/analytics.notify.cve/body.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Security alert notification in {{.Space.Data.Codebase.Name}}</title>
+  <style>
+    tr td ul li img {
+      max-width: 500px;
+    }
+
+    tr td pre {
+      overflow: auto;
+    }
+
+    a:hover {
+      text-decoration: underline !important;
+    }
+
+    table {
+    width:100%;
+    }
+
+    table, th, td {
+        border-collapse: collapse;
+    }
+
+    th, td {
+        padding: 15px;
+        border-bottom: 1px solid #ededed;
+        text-align: left;
+        color: #363636
+    }
+
+    th {
+      font-weight: 500;
+    }
+
+    td {
+      color: #72767B;
+      padding-top: 20px;
+      padding-bottom: 30px;
+    }
+  </style>
+</head>
+
+<body style="
+        margin: 10px;
+        padding: 0; 
+        background-color: #f9f9f9;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 14px;
+        font-weight: lighter;
+        line-height: 1.2;">
+  <div style="
+        min-height: 300px;
+        width: 100%;
+        max-width: 800px;
+        margin: 0 auto;
+        padding: 20px;
+        border: 1px solid #d7d7d7;
+        border-radius: 4px;
+        background-color: #fff;
+        box-shadow: 0 2px 4px #d7d7d7;">
+    <div style="
+          width: 100%;">
+      <img src="{{.webURL}}/_openshiftio/assets/images/openshift_logo_black.png" width="25%" height="45px" />
```

Comment:
this URL is still in issue in all of the templates - I'd leave it for the time being, but know that it will be blank.

Reply:
yes it gave 404, can we pick from some other resource or what would be the fix for it?

Reply:
The resource is actually in the repo, at the location listed, but it is not being served through openshift.io for some reason. It has yet to be figured out (it is a build/deployment of resources issue).

The only one that appears to be available via the server is the white version of the logo.

I'll see about updating the header of the templates to match the About modal, which is a dark background with the white logo.

Reply:
The resource is actually in the repo, at the location listed, but it is not being served through openshift.io for some reason. It has yet to be figured out (it is a build/deployment of resources issue).

At this time, my guess is that the hashes that are added to the image by the build are not recognized by the email templates, so the images are not found.

Reply:
Also, from what I remember when Aslak and I first created these, images cannot be added directly to the templates. They need to be available as an external resource in order to appear in the templates.

Reply:
Thanks @mindreeper2420 , let me if changes are fine or if you have any more suggestion/inputs.